server/resource_server/controller/controller.go

```diff
@@ -258,7 +259,7 @@ func VerifyEmailHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	response := utils.BuildResponse(true, "OK!", "Verification email sent")
+	response := utils.BuildResponse(w, http.StatusOK, "Verification email sent", utils.EmptyObj{})
```

Comment:
lets just create a new function for response body without data field instead of putting `utils.EmptyObj{}` everywhere

Reply:
The reason is that we need to be consistent with the response builder function, and not just create new function for each cases, it's just some more processing just to do the thing which can be done without cost of memory/processing, so that is why I used directly the `utils.EmptyObj{}`, because in some cases, we do need to return some data but when there is no data, just passing the EmptyObj struct. In this way we use the same response builder for all of our controllers and don't need to create extra function for handling a case which is being done in a simple way, what do you think?

Reply:
I don't have the experience to say either way. In this circumstance, we can maintain the status quo, (using `utils.EmptyOjb{}`) and, as time goes on, submit to a refactor if we find we're using it constantly. That is, in the future we can have two functions: 

```
utils.BuildResponse( ... )
utils.BuildResponseWithNoBody( ...  )
```

Reply:
IMO It is simpler to have multiple function rather than input the utils.EmptyObj{} because we only write the utils function once and use it everywhere. Put the function below the BuildResponse function so we can easily aware if we need to make any changes to the response body builder